test/index.test.js

```diff
@@ -57,4 +59,286 @@ describe('react-ga', function() {
+                                            eventValue: 10,
+                                            hitType: 'event'
+                                          }]
+                                ]);
+    });
+
+    it('should reject a non-numeric value value', function() {
+      ga.initialize('foo');
+      ga.event( { category: 'Test', action: 'Send Test', value: 'millions' } );
+      console.warn.args.should.eql([[
+        '[react-ga]', 'Expected `args.value` arg to be a Number.'
+      ]]);
+      getGaCalls().should.eql([ [ 'create', 'foo', 'auto' ],
+                                [ 'send', { eventAction: 'Send Test',
+                                            eventCategory: 'Test',
+                                            hitType: 'event'
+                                          }]
+                                ]);
+    });
+
+    it('should record a nonInteraction value', function() {
+      ga.initialize('foo');
+      ga.event( { category: 'Test', action: 'Send Test', nonInteraction: true } );
+      getGaCalls().should.eql([ [ 'create', 'foo', 'auto' ],
+                                [ 'send', { eventAction: 'Send Test',
+                                            eventCategory: 'Test',
+                                            nonInteraction: true,
+                                            hitType: 'event'
+                                          }]
+                                ]);
+    });
+
+    it('should reject a non-boolean nonInteraction value', function() {
+      ga.initialize('foo');
+      ga.event( { category: 'Test', action: 'Send Test', nonInteraction: 'yeahsure' } );
+      console.warn.args.should.eql([[
+        '[react-ga]', '`args.nonInteraction` must be a boolean.'
+      ]]);
+      getGaCalls().should.eql([ [ 'create', 'foo', 'auto' ],
+                                [ 'send', { eventAction: 'Send Test',
+                                            eventCategory: 'Test',
+                                            hitType: 'event'
+                                          }]
+                                ]);
+    });
+  });
+
+  /**
+   * outboundLink()
+   */
+
+  describe('outboundLink()', function() {
+
+    it('should record an outboundLink event', function() {
+      var callbackSuccess = false;
+      ga.initialize('foo');
+      ga.outboundLink( { label: 'Test Click' }, function () {
+        callbackSuccess = true;
+      } );
+      getGaCalls().should.eql([ [ 'create', 'foo', 'auto' ],
+                                [ 'send', { eventAction: 'Click',
+                                            eventCategory: 'Outbound',
+                                            eventLabel: 'Test Click',
+                                            hitCallback: undefined,
+                                            hitType: 'event'
+                                          }]
+                                ]);
+      callbackSuccess.should.eql(true);
```

Comment:
Another way of doing this is by using a [`sinon.spy()`](http://sinonjs.org/docs/#sinonspy); it's not particularly useful here because you're just testing whether the function has been called, but once you start wanting to test more things, e.g. the arguments passed to the function, sinon starts becoming more useful.